### DISCUSSION TITLE
I get notified of every single post. How to stop that?

### DISCUSSION THREAD

**Discussion by elSolus** [elementary/os#537] [Q&A] [answered]
Hi. I get notified via email about every single post and comment on every single repository related to elementary. If I started a post/issue, I understand that I get notified about that but why ALL? What did I do? Please help me stop it.

**Comment by tintou** [comment, marked as answer]
I've no idea what you did here, but you can unsubscribe from here https://github.com/watching

**Reply by elSolus** [reply]
Hey, @tintou Thanks a lot... I now remember I did that about a month ago. I'm new to GitHub so.. but thanks a ton again for the quick reply.

**Comment by davidmhewitt** [comment]
Have a look at what GitHub repositories you're "watching" here: https://github.com/watching

Are there a bunch of elementary ones in there that you don't want/need to be watching? I'm not sure how you would have ended up subscribed to them all, but that should be where you can unsubscribe.

**Reply by elSolus** [reply]
Hey, @davidmhewitt. Thanks a ton for the quick reply. It seems to fix it.
